package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@getinsomnia/node-libcurl",
-  "version": "2.31.5-alpha.1",
+  "version": "2.33.5-alpha.1",
```

Comment:
this is not needed

Reply:
im using it as a electron version signifier because the .electron-version approach would yield inconsistent npm behavior

Reply:
ok